strimzi-kafka-operator.yaml

```diff
@@ -0,0 +1,153 @@
+      wget -qO- "https://github.com/koalaman/shellcheck/releases/download/v${{vars.shellcheck_version}}/shellcheck-v${{vars.shellcheck_version}}.linux.${{build.arch}}.tar.xz" -O shellcheck.tar.xz
+
+      # Perform shasum check based on architecture
+      if [[ "${{build.arch}}" == "aarch64" ]]; then
+          echo "${{vars.shellcheck_aarch64_shasum512}}  shellcheck.tar.xz" | shasum -a 512 -c
+      elif [[ "${{build.arch}}" == "x86_64" ]]; then
+          echo "${{vars.shellcheck_x86_64_shasum512}}  shellcheck.tar.xz" | shasum -a 512 -c
+      else
+          echo "Unsupported architecture: ${{build.arch}}"
+          exit 1
+      fi
+
+      # Extract the tarball
+      tar -xJf shellcheck.tar.xz
+
+      # Copy shellcheck binary to /usr/bin/
+      cp "shellcheck-v${{vars.shellcheck_version}}/shellcheck" /usr/bin/
+
+
+      for i in mockkube config-model certificate-manager operator-common; do
+        make -C $i MVN_ARGS="-B -DskipTests -Dmaven.javadoc.skip=true" java_build
+        make -C $i MVN_ARGS="-DskipTests -Dmaven.javadoc.skip=true -e -V -B" java_install
+      done
+
+      mvn clean
+      mvn install -B -DskipTests -Dmaven.javadoc.skip=true
+      make MVN_ARGS="-B -DskipTests -Dmaven.javadoc.skip=true" java_build
+      make MVN_ARGS="-DskipTests -Dmaven.javadoc.skip=true  -e -V -B" java_install
+
+      # Set the source and destination directories
+      SRC_DIR="./docker-images/artifacts/binaries"
+      DEST_DIR="opt/strimzi"
+      mkdir -p "${{targets.contextdir}}"/$DEST_DIR/bin
+      mkdir -p "${{targets.contextdir}}"/$DEST_DIR/lib
+      install -Dm755 ./docker-images/operator/scripts/launch_java.sh ${{targets.contextdir}}/$DEST_DIR/bin/launch_java.sh
+
+
+      # List of file name patterns, space-separated
+      file_patterns="topic-operator user-operator cluster-operator kafka-init"
+
+      # Iterate over each pattern and match the corresponding zip files
+      for file in $file_patterns; do
+          # Find the zip file that matches the pattern
+          zipfile=$(ls "$SRC_DIR"/${file}-*-dist.zip 2>/dev/null)
+
+          # Check if the zip file exists
+          if [[ -f "$zipfile" ]]; then
+              # Extract the zip file into the destination directory
+              unzip -o "$zipfile" -d tmp
+
+              cp -r tmp/lib/* "${{targets.contextdir}}"/$DEST_DIR/lib/
+              cp -r tmp/bin/* "${{targets.contextdir}}"/$DEST_DIR/bin/
+
+              # Check if the unzip command was successful
+              if [[ $? -eq 0 ]]; then
+                  echo "Successfully unzipped $zipfile to $DEST_DIR"
+              else
+                  echo "Failed to unzip $zipfile"
+              fi
+          else
+              echo "No zip file found for pattern: ${file}-*-dist.zip"
+          fi
+      done
+
+update:
+  enabled: true
+  github:
+    identifier: strimzi/strimzi-kafka-operator
+
+test:
+  pipeline:
+    - name: check folders are not empty
+      runs: |
+        #!/bin/bash
```

Comment:
This doesn't work in tests. We'd do something like:

````
bash -c '
test contents
'
````

Reply:
actually we don't need this one, let's remove it